### PR TITLE
VIDEO: Fix crash when closing an MKV stream

### PR DESCRIPTION
**Pull request by ccawley2011**

### CODE REVIEW

File: video/mkv_decoder.cpp [hunk 1]
```diff
@@ -238,14 +237,18 @@ bool MKVDecoder::loadStream(Common::SeekableReadStream *stream) {
...
 	_frameCount = _pBlock->GetFrameCount();
+	_fileStream = stream;
 
 	return true;
 }
 
 void MKVDecoder::close() {
 	VideoDecoder::close();
 
+	delete _fileStream;
```
Comment (lephilousophe) [inline video/mkv_decoder.cpp:248]: To be really clean, reader should be deleted before as it's created after and depends on fileStream

**Comment by sev-** [comment]
@ccawley2011 Do you still plan to do the proposed code adjustment?

**Comment by lephilousophe** [comment]
This change is completely optional though

**Comment by sev-** [comment]
Okay, merging then. Thanks!

**Pull request merged**